### DISCUSSION TITLE
Resolve entrypoint from import_map.json?

### DISCUSSION THREAD

**Discussion by anishkny** [denoland/deno#16716] [General]
Hi folks - first off kudos on all the excellent work on Deno so far! 🎉🎉🎉 

I am trying to run [`lume`](https://lume.land/) locally via `import_map.json` thus:

```bash
$ cat import_map.json
{
  "imports": {
    "lume/": "https://deno.land/x/lume@v1.13.0/"
  }
}

$ deno run -A --unstable lume/cli.ts
error: Module not found "file:///Users/anish/workspace/lume-test/lume/cli.ts".
```

The suggested way in Lume docs is to specify deno.json:

```json
{
  "importMap": "./import_map.json",
  "tasks": {
    "lume": "echo \"import 'lume/cli.ts'\" | deno run --unstable -A -"
  }
}
```

But this is ugly.

Can we have an option to make deno resolve entry point CLI script (e.g. `lume/cli.ts`) from `import_map.json`? That would be awesome!

**Comment by jupegarnica** [comment]
$ deno run --import-map=import_map.json -A --unstable lume/cli.ts

**Reply by bartlomieju** [reply]
This is the correct solution

**Reply by anishkny** [reply]
Sorry. Doesn't work.

```bash
$ deno run --import-map=import_map.json -A --unstable lume/cli.ts
error: Module not found "file:///Users/anish/workspace/lume-test/lume/cli.ts".
```

**Reply by bartlomieju** [reply]
Err, sorry I misread what's going on here. This is tracked in https://github.com/denoland/deno/issues/14066, PRs to fix the behavior are welcome
